<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="Memtab">
       <attribute name="title">
        <string>内存使用</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="8">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="9">
         <widget class="QProgressBar" name="swapBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="9">
         <widget class="QChartView" name="memChart"/>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="Swapfree">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QLabel" name="Memuse">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Use:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Free:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="9">
         <widget class="QProgressBar" name="memBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Free:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="6">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Use:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="7">
         <widget class="QLabel" name="SwapUse">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="Memtotal">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="Memfree">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item row="4" column="8">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="Swaptotal">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Swap</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CPUtab">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>CPU使用</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>CPU1:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="cpu1Bar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>CPU2:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="cpu2Bar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>CPU曲线</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="5">
            <widget class="QLabel" name="yellow">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_19">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>CPU1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="red">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="7">
            <widget class="QChartView" name="cpuChart"/>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_20">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>CPU2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Protab">
       <attribute name="title">
        <string>进程管理</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Running:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="10">
         <widget class="QPushButton" name="btn_ShutDown">
          <property name="text">
           <string>关机</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QLabel" name="label_Zombie">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_Running">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Zombie:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Sleeping:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_Sleeping">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_Stopped">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Stopped:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="9">
         <widget class="QPushButton" name="btn_killProc">
          <property name="text">
           <string>结束进程</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="QPushButton" name="btn_viewProc">
          <property name="text">
           <string>详细信息</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_Total">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Idle:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="label_Idle">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="11">
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>state</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ppid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>priority</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Memory(MB)</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>查找进程</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Systab">
       <attribute name="title">
        <string>系统信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>主机名：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="hostname">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>启动时间:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="StartTime">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>持续运行：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="DualTime">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>系统版本号：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="Version">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>CPU基本信息</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="cpuList">
             <property name="textInteractionFlags">
              <set>Qt::NoTextInteraction</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
